www/docs/style/sass/pages/_mp.scss

```diff
@@ -638,6 +638,16 @@ html.js {
   }
 }
 
+.vote-description__covid {
+    display: block;
+    color: #757470;
+    text-decoration: none !important; // override .person-panels .panel a
```

Comment:
I don't understand this line, because there's no text decoration applied to a `p`?

Reply:
I'll take this out - it was because I was copying the style for the evidence section.